src/components/src/ImageContainer/image.container.tsx

```diff
@@ -84,13 +85,13 @@ const ImageContainer: React.FC<ImageContainerProps> = (props) => {
             type={`image/${type}`}
           />
         ))}
-        <img className={imgClassName} alt={imgAlt} src={fallbackImgUrl} key={fileName} onLoad={onLoadData} onError={() => handlePictureError()} />
+        <LazyImage className={imgClassName} alt={imgAlt} src={fallbackImgUrl} key={fileName} onLoad={onLoadData} onError={() => handlePictureError()} />
```

Comment:
Lazy loading won't work when SrcSets are enabled.  We may need to include the <picture> element with the IntersectionObserver